cleanlab/filter.py

```diff
@@ -236,50 +235,42 @@ def multiclass_crossval_predict(pyx, labels):
     return pred
 
 
-def get_noise_indices(
-        s,
-        psx,
-        inverse_noise_matrix=None,
-        confident_joint=None,
-        frac_noise=1.0,
-        num_to_remove_per_class=None,
-        prune_method='prune_by_noise_rate',
-        sorted_index_method=None,
-        multi_label=False,
-        n_jobs=None,
-        verbose=0,
-):
-    """Returns the indices of most likely (confident) label errors in s. The
+def find_label_issues(labels, psx, confident_joint=None, frac_noise=1.0,
+                      num_to_remove_per_class=None, filter_by='prune_by_noise_rate',
+                      return_indices_ranked_by=None, multi_label=False, n_jobs=None, verbose=0):
+    """By default, this method returns a boolean mask for the entire dataset where True represents
+    a label issue and False represents an example that is confidently/accurately labeled.
+
+    You can return ONLY the indices of the label issues in your dataset, by setting
+    return_indices_ranked_by = {`prob_given_label`, `normalized_margin`}.
+
     number of indices returned is specified by frac_of_noise. When
     frac_of_noise = 1.0, all "confident" estimated noise indices are returned.
     * If you encounter the error 'psx is not defined', try setting n_jobs = 1.
 
+    WARNING! frac_noise and num_to_remove_per_class parameters are only supported when filter_by
+    is either 'prune_by_noise_rate', 'prune_by_class', or 'both'. They are not supported for methods
+    'confident_learning' or 'argmax_not_equal_given_label'. TODO.
```

Comment:
`frac_noise` and `num_to_remove_per_class` seem like secondary args that should appear after more important args like: `filter_by`, `return_indices_ranked_by`.

Reply:
Great point!